pkg/config/config.go

```diff
@@ -56,7 +59,8 @@ func ParseCliArgs() Config {
 	flag.BoolVar(&config.DeleteLocalData, "delete-local-data", getBoolEnv(deleteLocalDataConfigKey, true), "If true, do not drain pods that are using local node storage in emptyDir")
 	flag.StringVar(&config.KubernetesServiceHost, "kubernetes-service-host", getEnv(kubernetesServiceHostConfigKey, ""), "[ADVANCED] The k8s service host to send api calls to.")
 	flag.StringVar(&config.KubernetesServicePort, "kubernetes-service-port", getEnv(kubernetesServicePortConfigKey, ""), "[ADVANCED] The k8s service port to send api calls to.")
-	flag.IntVar(&config.PodTerminationGracePeriod, "grace-period", getIntEnv(podTerminationGracePeriodConfigKey, -1), "Period of time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used.")
```

Comment:
Let's be good release stewards and provide this functionality in the 1.x release series. Simply deprecate the original `--grace-period` CLI option and note its rename to `--pod-termination-grace-period`.

Reply:
here's some simple testing:  https://gist.github.com/bwagner5/b18ddb5576b6d6dad13b5bd13dc1905d